SConstruct

```diff
@@ -1626,26 +1647,58 @@ if env['matlab_toolbox'] == 'y':
 # /usr/local because of dist-packages vs site-packages
 env['debian'] = any(name.endswith('dist-packages') for name in sys.path)
 
+# Check whether Cantera should be installed into a conda environment
+if conda_prefix is not None:
+    if env["layout"] != "conda":
+        use_conda = not selected_options & \
+            {"layout", "prefix", "python_prefix", "python_cmd"}
+        use_conda &= sys.executable.startswith(conda_prefix)
+        if use_conda:
+            env["layout"] = "conda"
+            logger.info(
+                f"Using conda environment as default 'prefix': {env['prefix']}")
```

Comment:
I don't think the `prefix` will have been replaced at this point, right? That happens in the next block